build-scripts/build_llvm.py

```diff
@@ -112,17 +112,11 @@ def build_llvm(llvm_dir, platform, backends, projects, use_clang=False, extra_fl
     ]
 
     # if not on ARC platform, but want to add expeirmental backend ARC as target
-    if platform != "ARC" and "ARC" in backends: 
+    if platform != "ARC" and "ARC" in backends:
         LLVM_TARGETS_TO_BUILD.extend(
             LLVM_EXTRA_COMPILE_OPTIONS["arc"]
         )
 
-    if platform != "Xtensa" and "Xtensa" in backends:
```

Comment:
@lum1n0us @TianlongLiang Is this mean that we can only build llvm with xtensa support on a xtensa host PC? As far as I known, there have no such platform